deploy/rbac/role_binding.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 subjects:
 - kind: ServiceAccount
   name: metal3-baremetal-operator
-  namespace: metal3
```

Comment:
don't change this, it breaks the baqckwards compatibility. All changes are done through kustomize